### PR TITLE
Refactor code snippet directive to have clearer variable names.

### PR DESCRIPTION
**Pull request by seanlip**
This is a follow-up to #591, specifically [this comment](https://github.com/google/tie/pull/591#pullrequestreview-129026731).

### CODE REVIEW

File: client/question/components/CodeSnippetDirective.js [hunk 1]
```diff
@@ -23,11 +23,11 @@ tie.directive('codeSnippet', [function() {
       getContent: '&content'
     },
     template: `
-      <span ng-repeat="line in abbreviatedSnippetLines track by $index">
+      <span ng-repeat="line in snippetLines track by $index">
         <span class="tie-code-snippet-line">{{line}}</span>
         <br>
       </span>
-      <span ng-if="abbreviatedSnippetLines.length > MAX_NUM_LINES_IN_ABBREVIATED_SNIPPET">
+      <span ng-if="codeLines.length > MAX_NUM_LINES_IN_SNIPPET">
```
Comment (rabidbit) [inline client/question/components/CodeSnippetDirective.js]: Just to confirm, abbreviatedSnippetLines is replaced by codeLines here but it's replaced by snippetLines elsewhere?

Comment (seanlip) [reply client/question/components/CodeSnippetDirective.js]: Yes, that was intended; I think the old code was less clear. Does this version make sense to you (logically)?

File: client/question/components/CodeSnippetDirective.js [hunk 2]
```diff
@@ -50,31 +50,30 @@ tie.directive('codeSnippet', [function() {
          */
         $scope.isModalOpen = MonospaceDisplayModalService.isDisplayed;
 
-        // The maximum number of lines to show in the abbreviated code snippet.
-        $scope.MAX_NUM_LINES_IN_ABBREVIATED_SNIPPET = 3;
+        // The maximum number of lines to show in a code snippet.
```
Comment (rabidbit) [inline client/question/components/CodeSnippetDirective.js]: Is context lost when we drop "abbreviated"? This is in reference to user code that is > 3 lines in the feedback window and is therefore truncated, right? Would this cause confusion with code snippets that are not truncated (e.g., short code or code displayed in a detailed code pop-up/modal)?

File: client/question/components/CodeSnippetDirective.js (hunk 1 above)
Comment (rabidbit) [reply client/question/components/CodeSnippetDirective.js]: I see. In that case, it seems the older code was actually different? I think the newer version makes sense.

File: client/question/components/CodeSnippetDirective.js (hunk 2 above)
Comment (rabidbit) [reply client/question/components/CodeSnippetDirective.js]: I see. I agree that the two terms sound redundant. As long as the convention of calling truncated code "snippet" is consistent throughout, SGTM.

File: client/question/components/CodeSnippetDirective.js (hunk 1 above)
Comment (seanlip) [reply client/question/components/CodeSnippetDirective.js]: Yes, that's right. Thanks!

**Comment by seanlip** [comment]
Thanks for the review!

**Pull request merged**